bitnami/redis/values.yaml

```diff
@@ -1000,6 +1008,14 @@ pdb:
   ## @param pdb.maxUnavailable Max number of pods that can be unavailable after the eviction
   ##
   maxUnavailable:
+## Autoscaling configuration
```

Comment:
Could you move all of those values inside the replica section, please?. I mean, the autoscaling is going to work only for the replicas and right now the values are in a section shared with the master and the replicas.

Reply:
Done